frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,216 @@
+        </v-card>
+        <v-card v-if="isReferenceLetterRequired" variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col cols="7" md="3" lg="2">
+              <AppLabel>Upload Documents</AppLabel>
+            </v-col>
+            <v-col>
+              <v-tooltip
+                content-class="tooltip"
+                text="A letter from a community representative or agency that has knowledge about the sector and can demonstrate how the society provides services to support the local community."
+                max-width="300px">
+                <template #activator="{ props }">
+                  <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
+                </template>
+              </v-tooltip>
+            </v-col>
+          </v-row>
+          <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+            <AppDocumentUpload
+              v-model="model.documentsToUpload"
+              :uploaded-documents="model.uploadedDocuments"
+              :document-type="DOCUMENT_TYPES.COMMUNITY_LETTER"
+              :document-label="'Letter of Reference for Community Support'"
+              entity-name="accounts"
+              :disabled="false"
+              :loading="loading"
+              :rules="[...rules.required]"
+              @delete-uploaded-document="deleteUploadedDocument"></AppDocumentUpload>
+            <div v-if="!areDocumentsComplete">
+              <v-row class="my-5">
+                <v-icon size="large" class="alert-icon pb-1 ml-5">mdi-alert-circle</v-icon>
+                <p class="text-error ml-2">You must upload a Community Support Letter</p>
+              </v-row>
+            </div>
+          </v-card>
+        </v-card>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col class="">
+              <AppLabel>Does your facility's not-for-profit have an open membership?</AppLabel>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col>
+              <v-radio-group v-model="model.openMembership" :readonly="false" :rules="[...rules.required]" hide-details>
+                <v-row no-gutters>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="Yes" :value="1"></v-radio>
+                  </v-col>
+                  <v-col cols="12" sm="2" lg="1">
+                    <v-radio label="No" :value="0"></v-radio>
+                  </v-col>
+                </v-row>
+              </v-radio-group>
+            </v-col>
+          </v-row>
+        </v-card>
+
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col class="">
```

Comment:
do we need v-row and v-col here? probably just AppLabel and v-radio-group should be enough

Reply:
done